Templates/DefaultProject/Template/export.bat

```diff
@@ -0,0 +1,47 @@
+@echo off
+REM --------------------------------------------------------------------------------------------------
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM 
+REM --------------------------------------------------------------------------------------------------
+
+REM This packaging script simplifies the project export command by defaulting the required and important
+REM arguments to the values based on:
+REM     --project-path %O3DE_PROJECT_PATH%      <- The path to this project for exporting
+REM     --tools-build-path %TOOLS_BUILD_PATH%   <- The location of the tools/editor build to make sure all the tools necessary 
+REM                                                for packaging are built and available (if applicable)
+REM     -out %OUTPUT_PATH%                      <- The output location for the exported project
+REM
+REM Feel free to adjust any of the arguments as necessary. For more information about the project export command, type in the 
+REM following command from the engine root
+REM
+REM %O3DE_PATH%\scripts\o3de.bat export-project -es ExportScripts\export_source_built_project.py --script-help
+REM
+REM To see the default options the export-project command will use other than the above required override arguments, you can view/edit
+REM the parameters with the export-project-configure command
+REM
+REM %O3DE_PATH%\scripts\o3de.bat export-project-configure --help
+REM
+REM To view the settings for the current project:
+REM
+REM %O3DE_PATH%\scripts\o3de.bat export-project-configure -p %O3DE_PROJECT_PATH% --list
+REM
+REM Note: The location of the engine (O3DE_PATH) is hardcoded to the location of the engine that was used to generate 
+REM       this project. The engine path must reflect the path to the engine on the local machine.
+
+set O3DE_PATH=${EnginePath}
```

Comment:
Is ${EnginePath} the only part that might be different from machine to machine? 
For future: could we assume the .o3de/o3de_manifest.json location and read the project.json to find the engine?
Then users would be able to check-in this file and have it work across all workspaces.

Reply:
There was a discussion about it in an earlier comment